nuitka/plugins/standard/PySidePyQtPlugin.py

```diff
@@ -1387,6 +1387,17 @@ def __init__(self, qt_plugins, no_qt_translations):
 Support for PyQt6 is not perfect, e.g. Qt threading does not work, so prefer PySide6 if you can."""
         )
 
+    @staticmethod
```

Comment:
Now these are quite a few. I think I dislike having to provide the full name.

Reply:
Do you mean the full name of  DLLs or method?

Reply:
Nah, the OS specific suffix, like `.dylib`. I image that makes plugins enter the business of knowing platform specific DLL suffixes. These should instead be compared in a way that the DLL suffix is stripped and then a match is done. I see how for now, this is macOS only code, but as a plugin interface it will otherwise hinder abstraction.